ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -39,8 +39,8 @@
                 </span>
 
                 <!--Search Field-->
```

Comment:
And ``placeholder="Filter platforms & jobs"`` below needs to be changed to `"Quick filter"` also.